lib/util/md5-hash.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
```

Comment:
Another option is to move this function into `cli-engine.js`, as that's the only place it seems to be used.